hc/front/tests/test_add_channel.py

```diff
@@ -38,4 +38,30 @@ def test_instructions_work(self):
             self.assertContains(r, "Integration Settings", status_code=200)
 
     ### Test that the team access works
+    def test_teamaccess_works(self):
+        url = "/integrations/add/"
```

Comment:
Try the endpoint `hc-add-channel` with reverse

Reply:
Noted